TopMovies/API/MovieTarget.swift

```diff
@@ -18,7 +18,10 @@ class MovieTarget: StoreSubscriber {
   }
   
   enum Target {
-    case marvelMovies
+    case nowPlaying(page: Int = 1)
+    case popular(page: Int = 1)
+    case topRated(page: Int = 1)
+    case upcoming(page: Int = 1)
```

Comment:
To avoid using `Target.upcoming()` you can write:
```swift
var upcoming: Target { Target.upcomming(page: 1) }
```

Reply:
If I just add computed properties to the MovieTarget type or the RequestedCategory enumeration (new name instead of Target), anyway I have to instantiate the type, if I want to use its properties, how about making them static?)

Reply:
Exactly static, I just missed that keyword )